ocrd/ocrd/workspace.py

```diff
@@ -390,9 +396,9 @@ def add_file(self, file_grp, content=None, **kwargs):
             if "file_id" in kwargs:
                 kwargs["ID"] = kwargs.pop("file_id")
 
-            ret = self.mets.add_file(file_grp, **kwargs)
+            ret = self.mets.add_file(file_grp, **kwargs, content=content)
```

Comment:
Since we decided against having the METS server pass file data and removed any `content` being passed from it to `OcrdWorkspace.add_file` , the `is_remote` check is not necessary, f477ae1f8